packages/components/src/components/autocomplete/autocomplete.tsx

```diff
@@ -116,21 +143,21 @@ const Autocomplete = React.memo(props => {
         }
     }, [should_show_list, is_alignment_top, list_portal_id, filtered_items.length]);
 
-    const handleScrollStop = e => {
+    const handleScrollStop = (e: React.UIEvent<HTMLDivElement, UIEvent>) => {
         // pass onScrollStop func callback when scrolling stops
-        if (typeof props.onScrollStop !== 'function') return;
+        if (!props.onScrollStop) return;
 
-        const element = e.target;
+        const element = e.currentTarget;
```

Comment:
@jim-deriv just to confirm: why do we need this change?  )

Reply:
@maryia-deriv, because the `current` property apparently doesn't exist on  the  `UIEvent<HTMLDivElement, UIEvent>'` type

Reply:
@jim-deriv the thing is TS simply doesn't know it exists in target.😊 Please check out this article https://www.designcise.com/web/tutorial/how-to-fix-property-does-not-exist-on-type-eventtarget-typescript-error
e.target really works here. IMHO replacing it with e.currentTarget just to satisfy TS will need thorough QA check because currentTarget is very often not the same element as target, and we might get unexpected behavior.

Reply:
Yes @maryia-deriv, I'll check it out 🙂